examples/provider/server.go

```diff
@@ -12,8 +12,6 @@ import (
 )
 
 func main() {
```

Comment:
some changes that were needed in this example server to avoid some errors i was getting